movies/webpack.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = process.env.NODE_ENV === 'development'
```

Comment:
for me mode=development doesn't set `NODE_ENV` to `development`  if I run `npm run serve`

Reply:
Fixed